client/components/HomePage.vue

```diff
@@ -2,7 +2,7 @@
   <main>
     <section>
       <header>
-        <h1> The Future of Healthcare </h1>
+        <h1> Welcome @{{ $store.state.username }}  to The Future of Healthcare </h1>
```

Comment:
should we have commas here if this is an appositive?

Reply:
I like a mix of both – "Welcome @xxx, to the future of healthcare"

Reply:
Is that gramatically correct?

Reply:
https://www.brandeis.edu/writing-program/resources/students/academic/style-grammar/comma-rules.html

I think it's ok according to rule 2